ledger-service/http-json/src/main/scala/com/digitalasset/http/Endpoints.scala

```diff
@@ -498,7 +498,8 @@ class Endpoints(
     for {
       jwt <- eitherT(input(req)).bimap(identity[Error], _._1)
       users <- EitherT.rightT(
-        userManagementClient.listUsers(Some(jwt.value))
+        // TODO participant user management: Emulating no-pagination
+        userManagementClient.listUsers(Some(jwt.value), pageToken = "", pageSize = 10000)
```

Comment:
@victormueller-da @moritzkiefer-da @nickchapman-da Currently requesting fixed large enough pageSize hoping to retrieve all the users. Not sure whether this is good enough for this PR.